src/functional/resources/expected/summary.js

```diff
@@ -1 +1 @@
ions":0,"fromDate":"2017-10-04","toDate":"2017-10-05"},{"insertions":0,"deletions":0,"fromDate":"2017-10-05","toDate":"2017-10-06"},{"insertions":0,"deletions":0,"fromDate":"2017-10-06","toDate":"2017-10-07"},{"insertions":0,"deletions":0,"fromDate":"2017-10-07","toDate":"2017-10-08"},{"insertions":0,"deletions":0,"fromDate":"2017-10-08","toDate":"2017-10-09"},{"insertions":0,"deletions":0,"fromDate":"2017-10-09","toDate":"2017-10-10"},{"insertions":100,"deletions":23,"fromDate":"2017-10-10","toDate":"2017-10-11"},{"insertions":203,"deletions":41,"fromDate":"2017-10-11","toDate":"2017-10-12"},{"insertions":0,"deletions":0,"fromDate":"2017-10-12","toDate":"2017-10-13"},{"insertions":0,"deletions":0,"fromDate":"2017-10-13","toDate":"2017-10-14"},{"insertions":0,"deletions":0,"fromDate":"2017-10-14","toDate":"2017-10-15"},{"insertions":0,"deletions":0,"fromDate":"2017-10-15","toDate":"2017-10-16"},{"insertions":0,"deletions":0,"fromDate":"2017-10-16","toDate":"2017-10-17"},{"insertions":0,"deletions":0,"fromDate":"2017-10-17","toDate":"2017-10-18"},{"insertions":10,"deletions":8,"fromDate":"2017-10-18","toDate":"2017-10-19"},{"insertions":345,"deletions":132,"fromDate":"2017-10-19","toDate":"2017-10-20"},{"insertions":0,"deletions":2,"fromDate":"2017-10-20","toDate":"2017-10-21"},{"insertions":91,"deletions":7,"fromDate":"2017-10-21","toDate":"2017-10-22"},{"insertions":0,"deletions":0,"fromDate":"2017-10-22","toDate":"2017-10-23"},{"insertions":153,"deletions":390,"fromDate":"2017-10-23","toDate":"2017-10-24"},{"insertions":294,"deletions":27,"fromDate":"2017-10-24","toDate":"2017-10-25"},{"insertions":0,"deletions":0,"fromDate":"2017-10-25","toDate":"2017-10-26"},{"insertions":0,"deletions":0,"fromDate":"2017-10-26","toDate":"2017-10-27"},{"insertions":0,"deletions":0,"fromDate":"2017-10-27","toDate":"2017-10-28"},{"insertions":0,"deletions":0,"fromDate":"2017-10-28","toDate":"2017-10-29"},{"insertions":0,"deletions":0,"fromDate":"2017-10-29","toDate":"2017-10-30"},{"insertions":69,"deletions":82,"fromDate":"2017-10-30","toDate":"2017-10-31"},{"insertions":317,"deletions":34,"fromDate":"2017-10-31","toDate":"2017-11-01"},{"insertions":35,"deletions":12,"fromDate":"2017-11-01","toDate":"2017-11-02"},{"insertions":0,"deletions":0,"fromDate":"2017-11-02","toDate":"2017-11-03"},{"insertions":0,"deletions":0,"fromDate":"2017-11-03","toDate":"2017-11-04"},{"insertions":0,"deletions":0,"fromDate":"2017-11-04","toDate":"2017-11-05"},{"insertions":0,"deletions":0,"fromDate":"2017-11-05","toDate":"2017-11-06"},{"insertions":0,"deletions":0,"fromDate":"2017-11-06","toDate":"2017-11-07"},{"insertions":180,"deletions":97,"fromDate":"2017-11-07","toDate":"2017-11-08"},{"insertions":0,"deletions":0,"fromDate":"2017-11-08","toDate":"2017-11-09"},{"insertions":0,"deletions":0,"fromDate":"2017-11-09","toDate":"2017-11-10"},{"insertions":109,"deletions":60,"fromDate":"2017-11-10","toDate":"2017-11-11"},{"insertions":246,"deletions":296,"fromDate":"2017-11-11","toDat
```

Comment:
explore why the insertion and deletion becomes 0.

Reply:
explored